tests_python/resources/_debugger_case_sysexit_unhandled_break.py

```diff
@@ -0,0 +1,7 @@
+import time
```

Comment:
This test change was how I verified the 'unhandled_exc' addition in the sys.monitoring for caching the unhandled frame.